### ISSUE TITLE
Issues with Android 5.0 Lollipop 

### ISSUE THREAD

**Issue by rajparekh07** [AppsFlyerSDK/appsflyer-cordova-plugin#125]
# Report

## Plugin Version

ℹ 6.0.54

## On what Platform are you having the issue?

ℹ Android

## What did you do?

ℹ Cordova add plugin
      Cordova run build

## What did you expect to happen?

ℹ appsflyer.js to work on Android lollipop API 21

## What happened instead?

ℹ
```
AppsFlyer.prototype.setOneLinkCustomDomains = function (domains, successC, errorC) {
        exec(successC, errorC, 'AppsFlyerPlugin', 'setOneLinkCustomDomains', [...domains]);
    };
```
Unexpected token . in appsflyer.js

## Please provide any other relevant information.

Most likely the recent javascript changes are not compatible with older webview engines. 
```
[...domains] 
```
is not getting recognised.

**Comment by amit-kremer93** [comment]
Hi @rajparekh07 and thank you for opening this issue. i will check this issue and get back to you

**Comment by amit-kremer93** [comment]
Hi @rajparekh07. according to the docs: `Cordova AppsFlyer plugin version 4.4.0 and higher are meant to be used with cordova-android@7.0.0 and up`. but for now, i fixed this api in the latest release (6.1.10) to work with lower versions as well.
Please let me know if it works for you

**Comment by rajparekh07** [comment]
@amit-kremer93 Thanks for a quick response, 6.1.10 seems working as expected. Appreciate it.
